README.md

```diff
@@ -6,41 +6,45 @@ Keda Manager is an extension to the Kyma ecosystem that allows users to install
 
 ![a](./docs/assets/keda-overview.drawio.svg)
 
+For more information, see [Use Keda Manager to manage KEDA](/docs/keda-management.md).
+
 ### What is KEDA?
 
-KEDA is a flexible Event Driven Autoscaler for the Kubernetes workloads. It extends the Kubernetes autoscaling mechanisms with its own metric server and the possibility to make use of external event sources to make scaling decisions. To learn more about KEDA, see [KEDA documentation](https://keda.sh/docs/latest/concepts/).
+KEDA is a flexible Event Driven Autoscaler for the Kubernetes workloads. It extends the Kubernetes autoscaling mechanisms with its own metric server and the possibility to make use of external event sources for making scaling decisions. To learn more about KEDA, see the [KEDA documentation](https://keda.sh/docs/latest/concepts/).
 
-## Install
+## Install Keda module
```

Comment:
why is this one long doc and not separated into smaller documents (for example, installation instructions separately as a [tutorial](https://github.com/kyma-project/community/blob/main/templates/resources/tutorial.md)) ?